README.md

```diff
@@ -57,13 +57,13 @@ poetry run pre-commit install
 
 ## Documentation
 
-This package requires specific input parameters, for more detail on these parameters please refer to our [documentation for Web API](https://docs.smileidentity.com/server-to-server/python).
+This package requires specific input parameters, for more detail on these parameters please refer to our [documentation for Web API](https://docs.usesmileid.com/server-to-server/python).
 
-Please note that you will have to be a Smile Identity Partner to be able to query our services. You can sign up on the [Portal](https://portal.smileidentity.com/signup).
+Please note that you will have to be a Smile Identity Partner to be able to query our services. You can sign up on the [Portal](https://portal.usesmileid.com/signup).
 
 ## Getting Help
 
-For usage questions, the best resource is [our official documentation](https://docs.smileidentity.com). However, if you require further assistance, you can file a [support ticket via our portal](https://portal.smileidentity.com/partner/support/tickets) or visit the [contact us page](https://portal.smileidentity.com/partner/support/tickets) on our website.
+For usage questions, the best resource is [our official documentation](https://docs.usesmileid.com). However, if you require further assistance, you can file a [support ticket via our portal](https://portal.usesmileid.com/partner/support/tickets) or visit the [contact us page](https://portal.usesmileid.com/partner/support/tickets) on our website.

```

Comment:
That contact us link is pointing at the portal rather than the website.  I'm pretty sure the tickets link on the portal doesn't exist anymore since we're using Intercom... Should probably check and change that first!